src/Components/Topbar/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+export interface TopBarProps {
+  /** 왼쪽방향 아이콘과 제목 사이의 gap */
+  gap?: number;
+
+  /** Topbar Title content */
+  children?: React.ReactNode;
+}
+
+export const TopBar = ({ gap = 12, children }: TopBarProps) => {
+  const navigate = useNavigate();
+  return (
+    <TopbarWrapper gap={gap}>
+      <MoveBack onClick={() => navigate(-1)}>
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+          <path
+            d="M17.4396 24C17.2348 24 17.03 23.922 16.874 23.7656L6.21997 13.112C5.60677 12.4988 5.60677 11.5012 6.21997 10.8884L16.874 0.234422C17.1864 -0.0779779 17.6928 -0.0779779 18.0052 0.234422C18.3176 0.546822 18.3176 1.05322 18.0052 1.36562L7.37117 12L18.0056 22.6344C18.318 22.9468 18.318 23.4532 18.0056 23.7656C17.8492 23.922 17.6448 24 17.44 24H17.4396Z"
+            fill="black"
+            fill-opacity="0.65"
+          />
+        </svg>
+      </MoveBack>
+      <TopBarContent>{children}</TopBarContent>
+    </TopbarWrapper>
+  );
+};
+
+const TopbarWrapper = styled.div<{ gap?: number }>`
+  display: flex;
+  flex: 1 0 0;
+  max-width: 1224px;
+  width: 100%;
+  align-items: center;
+  gap: ${({ gap }) => `${gap}px`};
+`;
```

Comment:
```suggestion
const TopbarWrapper = styled.div<{ $gap?: number }>`
  display: flex;
  flex: 1 0 0;
  max-width: 1224px;
  width: 100%;
  align-items: center;
  gap: ${({ $gap }) => `${$gap}px`};
`;
```

다음과 같이 `gap`을 `$gap`으로 선언해서 의도치 않게 \<div\>에 `gap` prop을 전달하는 일을 방지할 수 있을 것 같아요!

> Think of [transient props](https://medium.com/@probablyup/introducing-transient-props-f35fd5203e0c) as a lightweight, but complementary API to shouldForwardProp. Because styled-components allows any kind of prop to be used for styling (a trait shared by most CSS-in-JS libraries, but not the third party library ecosystem in general), adding a filter for every possible prop you might use can get cumbersome.
> 
> Transient props are a new pattern to pass props that are explicitly consumed only by styled components and are not meant to be passed down to deeper component layers. Here's how you use them:

Reply:
반영하겠습니다.!!